RELEASE_NOTES.md

```diff
@@ -37,8 +43,8 @@ v1.0.0-alpha.X
   [#457](https://github.com/OpenAssetIO/OpenAssetIO/issues/457)
 
 - Renamed both `ManagerFactoryInterface` as well as the subclass
-  `PluginSystemManagerFactory` to `ManagerInterfaceFactoryInterface`
-  and `PluginSystemManagerInterfaceFactory`, respectively, to avoid
+  `PluginSystemManagerFactory` to `ManagerImplementationFactoryInterface`
```

Comment:
[consider][minor] 72 char

Reply:
I left this one, as it was really hard to read, and it only seems to be over by 1